src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ToolStripCustomTypeDescriptorTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class ToolStripCustomTypeDescriptorTests : IDisposable
+{
+    private readonly ToolStrip _toolStrip;
+    private readonly ToolStripCustomTypeDescriptor _descriptor;
+
+    public ToolStripCustomTypeDescriptorTests()
+    {
+        _toolStrip = new();
+        _descriptor = new(_toolStrip);
+    }
+
+    public void Dispose()
+        => _toolStrip.Dispose();
+
+    [Fact]
+    public void Constructor_InitializesInstance()
+    {
+        ToolStrip instance = _descriptor.TestAccessor().Dynamic._instance;
+        instance.Should().Be(_toolStrip);
+    }
+
+    [Fact]
+    public void GetPropertyOwner_ReturnsInstance()
+        => _descriptor.GetPropertyOwner(null).Should().Be(_toolStrip);
```

Comment:
```suggestion
    public void GetPropertyOwner_ReturnsInstance() =>
        _descriptor.GetPropertyOwner(null).Should().Be(_toolStrip);
```

Per the style guide, we are breaking lines after the `=>`

Reply:
Apologies for the oversight. I reviewed the style guide again and will make sure to align with the standard going forward. Thank you.